### DISCUSSION TITLE
Risposta ad avvisi/circolari

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#437] [Idee e proposte]
Buonasera, volevo sapere se fosse possibile inserire la possibilità di richiedere una risposta tramite casella di testo o una adesione in risposta ad avviso o circolare.

**Comment by trinko** [comment]
No, è previsto solo la possibilità di segnare la lettura dell'avviso/circolare.

**Reply by Lucaccc** [reply]
No, intendevo se potesse essere una richiesta da sviluppare per le prossime versioni

**Reply by trinko** [reply]
Penso proprio di no: le circolari e gli avvisi sono pensati per una comunicazione unidirezionale, dalla scuola a famiglie/alunni/docenti.

La gestione che proponi, mi pare di capire sia più vicina a quella delle richieste: ad esempio si potrebbe creare un modulo per l'adesione ad una data attività, e i genitori che vogliono aderire dovranno inviare il modulo di richiesta.

**Reply by Lucaccc** [reply]
Si, sarebbe un'ottima idea.
Il modello al quale facevo riferimento io è quello utilizzato da diversi registri, come ad esempio Classeviva, Nuvola,..
Comunque anche la sua è un'ottima idea.
